src/Collections-Strings/String.class.st

```diff
@@ -2580,6 +2580,13 @@ String >> withCRs [
 	^ self collect: [ :c | c = $\ ifTrue: [ Character cr ] ifFalse: [ c ]].
 ]
 
+{ #category : #'platform conventions' }
+String >> withInternalLineEndings [
+	"Answer a new instance where all occurrences of CRLF and LF are substituted with CR. Pharo internally uses CR for carriage return."
+
+	^ self withLineEndings: String cr
+]
+
```

Comment:
BTW, wasn't there a project to bring Character lf as the internal lineEnding like was done in Cuis?

Reply:
It would be awesome indeed :)